src/Accordion.php

```diff
@@ -27,7 +26,7 @@
  *
  * ```php
  * echo Accordion::widget()
- *     ->items([
+ *     ->witItems([
```

Comment:
I think the name could be left as is.

Reply:
All the methods are immutable so they must have the prefix with, break BC but then we have the package ready.

Reply:
```suggestion
 *     ->withItems([
```